src/services/search.ts

```diff
@@ -0,0 +1,45 @@
+import uFuzzy from '@leeoniya/ufuzzy';
+import { debounce as debounceLodash } from 'lodash';
```

Comment:
Not important, just curious to know why you renamed it to `debouceLodash`.

Reply:
This was pulled from grafana/grafana, it was probably being used in a context where the debounce from debounce-promise was being used